### PR TITLE
Ilhan' code study

### PR DESCRIPTION
**Pull request by BobIT37**
Hey, Jason I have completed my study. FYI

### CODE REVIEW

File: src/main/java/SauceLabs/deneme.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package SauceLabs;
+
+public class deneme {
+	
+	
+	//empty
```
Comment (BobIT37) [inline src/main/java/SauceLabs/deneme.java:6]: Hey ilhan fizzier this part and add collection concept List

Comment (BobIT37) [reply src/main/java/SauceLabs/deneme.java:6]: Ok I fixed and test is running now check it again

Comment (BobIT37) [reply src/main/java/SauceLabs/deneme.java:6]: Ok looks fine push it

Comment (SebahattinG) [reply src/main/java/SauceLabs/deneme.java:6]: Good job.

**Pull request merged**